php/sync/class-sync-queue.php

```diff
@@ -188,9 +188,14 @@ public function load_hooks() {
 	 * @return mixed
 	 */
 	public function change_setting_state( $new_value, $current_value, $setting ) {
-		if ( $this->is_running() && 'on' === $current_value && 'off' === $new_value ) {
-			$this->shutdown_queue();
-			$setting->add_error_notice( 'disabled_sync', 'Bulk sync has been disabled.', 'warning' );
+		// shutdown queues if needed.
+		if ( 'on' === $current_value && 'off' === $new_value ) {
+			if ( $this->is_running() ) {
+				$this->shutdown_queue( 'queue' );
+				$setting->add_error_notice( 'disabled_sync', 'Bulk sync has been disabled.', 'warning' );
```

Comment:
@DavidCramer 

I'm noticing here a problem, as `add_error_notice()` doesn't exist. Also, this message is missing i18n.